src/Symfony/Component/Serializer/Mapping/ClassDiscriminatorFromClassMetadata.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Mapping;
+
+use Symfony\Component\Serializer\Mapping\Factory\ClassMetadataFactoryInterface;
+
+/**
+ * @author Samuel Roze <samuel.roze@gmail.com>
+ */
+class ClassDiscriminatorFromClassMetadata implements ClassDiscriminatorResolverInterface
+{
+    /**
+     * @var ClassMetadataFactoryInterface
+     */
+    private $classMetadataFactory;
+    private $mappingForMappedObjectCache = [];
```

Comment:
I don't think the short syntax for arrays has been accepted. Php-CS-fixer still has the long syntax specified in the config file

Reply:
But if we target PHP7+ we need to accept this :)

Reply:
I agree with you. But fabbot is still complaining. I think this rule should be removed from php_cs.dist and from fabbot, to permit new developments to use the short syntax. Would you like to open an issue / PR about this? At this point, according to this PR, they try to avoid merge conflicts.

Reply:
https://github.com/symfony/symfony/pull/24396 👍